core/transport/src/main/proto/proto/journal/file.proto

```diff
@@ -284,3 +284,19 @@ message UpdateUfsModeEntry {
   optional string ufsPath = 1;
   optional UfsMode ufsMode = 2 [default = READ_WRITE];
 }
+
+message LoadOptions {
+  optional int64 bandWidth = 1;
+}
+
+enum LoadState {
+  CREATED = 1;
+  FINISHED = 2;
+}
+
+message LoadDirectory {
+  optional int64 loadId = 1;
+  optional string srcFilePath = 2;
+  optional LoadOptions options = 3;
+  optional LoadState state = 4;
```

Comment:
Why do we need `LoadState` here? Isn't the state a return value decided by server?

Reply:
Initially wanted to use it for journaling the state of load. Looks like not necessary now, will remove

Reply:
Will delete this in this PR